app/modules/entities/components/layouts/authors_info_role.svelte

```diff
@@ -18,7 +18,7 @@
           { @const hasManyClaimValues = claims[prop].length > 15}
           {#each claims[prop] as claimValue}
             <AuthorDisplay
-              entity={authorsByUris[claimValue]}
+              {authorsByUris}
```

Comment:
I think that's a mistake: the component doesn't actually need the other entities data. I suspect you did that to work around an issue with components failing to update on changes? Doesn't using keyed each, something like ` {#each claims[prop] as claimValue (claimValue)}` fix the issue?

Reply:
yes, done in 4bed90370

Reply:
Then can this line be reverted to `entity={authorsByUris[claimValue]}`?

Reply:
i dont understand why it hasnt been pushed actually, heres another try in 23cb87de5